app/src/main/res/layout/activity_main.xml

```diff
@@ -4,15 +4,31 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@drawable/wallpaper_main_screen"
     tools:context=".MainActivity">
 
     <TextView
+        android:id="@+id/title_text"
+        style="@style/TitleStyleMainActivity"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:text="@string/main_activity_text_title"
+        android:layout_marginStart="@dimen/text_main_activity_margin_start"
+        android:layout_marginEnd="@dimen/text_main_activity_margin_end"
+        android:layout_marginTop="@dimen/text_main_activity_margin_top"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"/>
+
+    <Button
+        android:id="@+id/set_button"
```

Comment:
"set_button" I think is not a correct name, because if you have another button and set another thing. It is going to be ambiguous

Reply:
done, changed to button_main_activity_select_parking_space